packages/graphql-key-transformer/src/KeyTransformer.ts

```diff
@@ -454,22 +454,39 @@ export class KeyTransformer extends Transformer {
     if (this.isPrimaryKey(directive)) {
       const directiveArgs: KeyArguments = getDirectiveArguments(directive);
 
+      // check @model mutation argument to update schema or not
+      let shouldMakeCreate = true;
+      let shouldMakeUpdate = true;
+      let shouldMakeDelete = true;
+
+      if (ctx.featureFlags.getBoolean('skipOverrideMutationInputTypes', false)) {
+        const modelDirective = definition.directives.find(dir => {
```

Comment:
should just be `.find(dir => dir.name.value === 'model')`

.find() will return the object where the predicate is true

Reply:
Thanks!